content/manual/plugins-dev.md

```diff
@@ -119,11 +119,11 @@ a query comes in that is not a subdomain (or equal to) `example.org` the next pl
 
 Now, the world isn't perfect, and there are good reasons to "fallthrough" to the next middleware,
 meaning a plugin is only responsible for a *subset* of names within the zone. The first of these
-to appear was the *reverse* plugin that synthesizes PTR and A/AAAA responses (useful with IPv6).
+to appear was the *template* plugin that synthesizes various responses.
 
-The nature of the *reverse* plugin is such that it only deals with A/AAAA and PTR, and then only
-for a subset of the names. Ideally, you would want to layer *reverse* **in front of** another
-plugin such as *file* or *auto* (or even *proxy*). This means *reverse* handles some special
+The nature of the *fallthrough* plugin is such that it only deals with a specified record TYPE, and then only
```

Comment:
that's not true

Reply:
Good catch. Wrong thought written down. Should be fixed.